pkgs/tools/misc/promexplorer/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, nimPackages, fetchFromGitHub, fetchpatch }:
+nimPackages.buildNimPackage rec {
+  pname = "nimexplorer";
+  version = "0.0.3";
+  nimBinOnly = true;
+  src = fetchFromGitHub {
+    owner = "marcusramberg";
+    repo = "promexplorer";
+    rev = "v${version}";
+    hash = "sha256-q+m4+aWT3IiI/XGmAm7jrJAxRbzzHr+p58eiHqjEbV0=";
+  };
+
+  buildInputs = with nimPackages; [ illwill illwillwidgets ];
+
+  meta = with lib; {
+    description = "A simple tool to explore prometheus exporter metrics";
+    homepage = "https://github.com/marcusramberg/promexplorer";
+    license = licenses.mit;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ marcusramberg ];
+    mainProgram = "promexplorer";
```

Comment:
```suggestion
```

Reply:
Thanks for the review, just so I understand, do you suggest removing mainProgram because it's the same as the package name?

Reply:
> do you suggest removing mainProgram because it's the same as the package name?

Yes